src/Lykke.Service.Zcash.Api.AzureRepositories/History/HistoryRepository.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using AzureStorage;
+using AzureStorage.Tables;
+using Common;
+using Common.Log;
+using Lykke.AzureStorage.Tables;
+using Lykke.AzureStorage.Tables.Paging;
+using Lykke.Service.Zcash.Api.Core.Domain;
+using Lykke.Service.Zcash.Api.Core.Domain.Addresses;
+using Lykke.Service.Zcash.Api.Core.Domain.History;
+using Lykke.SettingsReader;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Lykke.Service.Zcash.Api.AzureRepositories.History
+{
+    public class HistoryRepository : IHistoryRepository
+    {
+        private INoSQLTableStorage<HistoryItemEntity> _historyStorage;
+        private INoSQLTableStorage<IndexEntity> _indexStorage;
+        private static string GetHistoryPartitionKey(ObservationCategory category, string address) => $"{Enum.GetName(typeof(ObservationCategory), category)}_{address}";
+        private static string GetHistoryRowKey(DateTime timestamp, string hash) => $"{timestamp.ToString("yyyyMMddHHmmss")}_{hash}";
```

Comment:
Use format specifier instead of ToString

Reply:
Fixed